h2o-r/h2o-package/R/models.R

```diff
@@ -2615,6 +2628,10 @@ h2o.negative_log_likelihood <- function(model) {
 #' }
 #' @export 
 h2o.average_objective <- function(model) {
+
+    warning_message <- paste("This objective function is calculated based on the simplified negative log likelihood ",
```

Comment:
`paste` concatenates and adds a space between the strings so this would end up with `...log likelihood<space><space>function...` You can use `paste0` instead. This little "bug" is in multiple r warning messages.